native/sapp-windows/external/sokol/sokol_app.h

```diff
@@ -3222,6 +3223,9 @@ typedef void  (GL_APIENTRY *PFN_glStencilFuncSeparate)(GLenum face, GLenum func,
 static PFN_glStencilFuncSeparate _sapp_glStencilFuncSeparate;
 typedef void  (GL_APIENTRY *PFN_glStencilOpSeparate)(GLenum face, GLenum sfail, GLenum dpfail, GLenum dppass);
 static PFN_glStencilOpSeparate _sapp_glStencilOpSeparate;
+void glStencilOpSeparate(GLenum face, GLenum sfail, GLenum dpfail, GLenum dppass) {
```

Comment:
Question: why wasn't this function already auto-generated in sokol_app.h - how does the generation work?

Wondering if I should add it manually or there's some automated way that you use to generate some of the functions in this file.

Reply:
All that functions were hand-written, just like you did for `glStencilOpSeparate`. 
In C function pointers was called directly with macroses, but for rust I did not found anything smarter than just handwrite a few function declarations.

Reply:
good to know, thanks!